tokio/src/net/mod.rs

```diff
@@ -46,6 +46,7 @@ cfg_net! {
         pub use tcp::socket::TcpSocket;
 
         mod udp;
+        #[doc(inline)]
         pub use udp::UdpSocket;
```

Comment:
Alternatively one could wrap the re-export here in another `cfg_net!`, but I thought `#[doc(inline)]` would be less confusing:

```suggestion
        cfg_net! {
            pub use udp::UdpSocket;
        }
````